packages/core/data-transfer/lib/providers/local-strapi-destination-provider/index.ts

```diff
@@ -92,6 +96,49 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
     return mapSchemasValues(schemas);
   }
 
+  getAssetsStream(): NodeJS.WritableStream {
+    if (!this.strapi) {
+      throw new Error('Not able to stream Assets. Strapi instance not found');
+    }
+
+    const assetsDirectory = path.join(this.strapi.dirs.static.public, 'uploads');
+    const backupDirectory = path.join(
+      this.strapi.dirs.static.public,
+      `uploads_backup_${Date.now()}`
+    );
+
+    fse.renameSync(assetsDirectory, backupDirectory);
+    fse.mkdirSync(assetsDirectory);
+
+    return new Writable({
+      objectMode: true,
+      async final(next) {
+        fse.rmSync(backupDirectory, { recursive: true, force: true });
```

Comment:
I think this will delete the original files in the case of "unable to rename backup -> uploads"

Reply:
Yes, you are right, I populated the error so that the final method isn't executed in case there is any error in that step, I hope this is what you had in mind